yt_dlp/extractor/youtube/pot/README.md

```diff
@@ -0,0 +1,305 @@
+# YoutubeIE PO Token Provider Framework
+
+As part of the YouTube extractor, we have a framework for providing PO Tokens programmatically. This can be used in plugins.
```

Comment:
pedant strikes again
```suggestion
As part of the YouTube extractor, we have a framework for providing PO Tokens programmatically. This can be used by plugins.
```